utility-grants/index.html

```diff
@@ -34,7 +34,7 @@ <h2>Spring 2025 Request for Proposals: Content-Addressed Data Tools</h2>
   <h3>RFP #2025-01: CAR Archive Explorers and Utilities</h3>
   <p>The CAR format (Content Addressable aRchives) makes data reliably portable by packaging up content with its cryptographic identifiers. This ensures the same data can be verified and used across different systems and platforms, making it a good building block for “credible exit” and data portability. Typically found in a file with a <code>.car</code> filename extension, you can think of them like TAR files that are designed for storing collections of content addressed data.</p>
   <p>Originally created as a way to create stable documents from large or dynamic IPLD graphs (including those graphs that populate “virtual directories” of files added to IPFS), CAR files can now be found in IPFS-based systems, Storacha, ATProto, and other open source projects. With growing interest in data independence – moving large datasets between storage providers, backing up personal content from hosted services, and maintaining data integrity across the web – there's a growing need for accessible tools that help people create, inspect, and manage their own CAR files.</p>
-  <p>Currently, there is limited stand-alone tooling for the CAR format. Prior art includes <a href="https://docs.ipfs.tech/reference/kubo/cli/#ipfs-dag-import">kubo</a>'s <code>dag import</code> and <code></code>dag export</code> CLIs, and Bluesky’s <a href="https://github.com/bluesky-social/indigo/blob/main/cmd/goat/repo.go">goat</a>, but they are mostly embedded within larger app-specific libraries and not ergonomic to use outside of that context. Standalone tools include <a href="https://www.npmjs.com/package/ipfs-car">ipfs-car</a> and <a href="https://github.com/ipld/go-car">go-car</a>.</p>
+  <p>Currently, there is limited stand-alone tooling for the CAR format. Prior art includes <a href="https://docs.ipfs.tech/reference/kubo/cli/#ipfs-dag-import">kubo</a>'s <code>dag import</code>, <a href="https://github.com/hsanjuan/caribdis">caribdis</a> and <code></code>dag export</code> CLIs, and Bluesky’s <a href="https://github.com/bluesky-social/indigo/blob/main/cmd/goat/repo.go">goat</a>, but they are mostly embedded within larger app-specific libraries and not ergonomic to use outside of that context. Standalone tools include <a href="https://www.npmjs.com/package/ipfs-car">ipfs-car</a> and <a href="https://github.com/ipld/go-car">go-car</a>.</p>
```

Comment:
caribdis is not embedded within a larger app, it is more similar to `go-car`. It has some extra features (concat) while being much smaller to peek through.

Reply:
oops merged before i could add another commit

Reply:
got to it in a later direct [commit](https://github.com/ipfs/implementations-grants/commit/3e89af0be4cac268869618c46134b27364fe80af)